packages/wonder-blocks-button/src/components/button-core.tsx

```diff
@@ -429,18 +416,18 @@ export const _generateStyles = (
                 borderStyle: "solid",
                 borderWidth: theme.border.width.secondary,
                 paddingInline: padding,
-                [":hover:not([aria-disabled=true])" as any]: hoverStyling,
-                [":focus-visible:not([aria-disabled=true])" as any]:
-                    focusStyling,
-                [":active:not([aria-disabled=true])" as any]:
-                    activePressedStyling,
+                ":hover": hoverStyling,
+                ":focus-visible": focusStyling,
```

Comment:
Will we move this to use the global focus styling later on? 😄

Reply:
That's correct, I didn't want to introduce too many visual changes in this PR, and I'm planning to cut a release with this change only for now. I checked in `webapp` and there are ~43 instances of `Button.light`, so I want to avoid having to include more changes than needed atm.